test/load/traders/random-trader.js

```diff
@@ -0,0 +1,46 @@
+const Promish = require('promish');
+const Trader = require('./trader');
+const strategyConfig = require('../random-strategy.json');
+
+class RandomTrader extends Trader {
+
+  start() {
+    this.isActive = true;
+    this.marketPrice = strategyConfig.price;
+    const loop = () => {
+      if (!this.isActive) {
+        this.promise = Promish.resolve();
+      } else {
+        const random = (Math.random() * 2) - 1; // generate random number from -1 to 1
+        if (random % 2 === 0) {
```

Comment:
This line will (almost) always be true.
Remove the if statement. Since random is from -1 to +1, we can just add to marketPrice